packages/aws-cdk-lib/aws-rds/lib/parameter-group.ts

```diff
@@ -70,6 +70,13 @@ export interface ParameterGroupProps {
    */
   readonly engine: IEngine;
 
+  /**
+   * The name of this parameter group.
+   *
+   * @default CloudFormation-generated name
```

Comment:
```suggestion
   * @default - CloudFormation-generated name
```
Something I learned recently, if the value immediately after a `@default` tag is not the actual value but rather a sentence describing the value, you need to separate it with a dash `-` or else it would think that `Cloudformation-generated` is the default value.